lib/src/dynamics/body.dart

```diff
@@ -555,8 +539,8 @@ class Body {
   /// @return the world velocity of a point.
   Vector2 getLinearVelocityFromWorldPoint(Vector2 worldPoint) {
     return Vector2(
-      -_angularVelocity * (worldPoint.y - _sweep.c.y) + _linearVelocity.x,
-      _angularVelocity * (worldPoint.x - _sweep.c.x) + _linearVelocity.y,
+      -_angularVelocity * (worldPoint.y - _sweep.c.y) + linearVelocity.x,
+      _angularVelocity * (worldPoint.x - _sweep.c.x) + linearVelocity.y,
     );
```

Comment:
how about something like
```
val radius = Vector2(_sweep.c.y - worldPoint.y, worldPoint.x - _sweep.c.x)
return _angularVelocity.times(radius) + linearVelocity
```

Reply:
this looks almost like a dot product as well, not sure

Reply:
Hmm, how can it be the dot product if it returns a vector and not a double?